composer.json

```diff
@@ -23,7 +23,7 @@
     "wikimedia/composer-merge-plugin": "1.4.1",
     "wpackagist-plugin/akismet": "4.*",
     "wpackagist-plugin/duplicate-post": "3.*",
-    "wpackagist-plugin/elasticpress":"3.2.*",
+    "wpackagist-plugin/elasticpress":"3.5.*",
```

Comment:
I guess for this plugin it may be better to play safe and not use *, as we have a lot of customizations that could break with even small changes to the plugin.

For now the latest tag that has a release on their repo is `3.5` (they add the first one of a new minor version without a patch version).

The tag for `3.5.1` already exists, but they didn't make a release for that. Maybe we should play safe and wait a bit until they (officially) release that version? There's usually a high chance that repos release some bug fixes in the first patch versions after a minor version bump, especially if the minor version has significant changes, which seems to be the case for `3.5`.

Reply:
Looking at their [Changelog](https://github.com/10up/ElasticPress/blob/develop/CHANGELOG.md), `3.5.1` seems a bug fixing release, so I think it's safe to use it.